.github/workflows/publish.yml

```
@@ -0,0 +1,64 @@
+name: Publish AiKit Packages to NPM
+
+on:
+  release:
+    types: [published]
+    tags:
```

actionlint (1.7.9)

"tags" filter is not available for release event. it is only for push event

(events)